### PR TITLE
Update lane preference, browser specific instructions and spelling fix.

### PR DESCRIPTION
**Pull request by sithrebel15**

### CODE REVIEW

**Comment by sithrebel15** [comment]
99151b527cccdbdd4839a1481770f5b2238236a4 and 
2dedb22217a87e8fbe596472f9bc504b829c9d5a add a preference for a lane with raining gold active.

File: autoPlay.js [hunk 1]
```diff
@@ -128,10 +130,46 @@ function goToLaneWithBestTarget() {
 			}
 		}
 	
+		//Prefer lane with raining gold, unless current enemy target is a treasure or boss.
+		if(lowTarget != ENEMY_TYPE.TREASURE && lowTarget != ENEMY_TYPE.BOSS ){
```
Comment (mouseas) [inline autoPlay.js:134]: lowTarget isn't a monster type, it's the slot in the current lane to target. This will malfunction.

**Comment by mouseas** [comment]
Can you please split the README changes into a separate pull request from the Raining Gold code changes? They're two different changes, unrelated.

**Comment by sithrebel15** [comment]
Too lazy to rebase closing and creating new.

**Pull request closed without merging**